packages/compiler/emitter-framework/types.ts

```diff
@@ -142,7 +142,7 @@ export type AssetTagFactory = {
   (value: string): AssetTagInstance;
 };
 
-export type CadlDeclaration = Model | Interface | Union | Operation | Enum | Scalar;
+export type TypeSpecDeclaration = Model | Interface | Union | Operation | Enum | Scalar;
```

Comment:
Let's export a deprecated alias for back compat. I think this feedback applies to any type name changes.